### DISCUSSION TITLE
Ideal desktop spec for using readfish

### DISCUSSION THREAD

**Discussion by jennieli421** [LooseLab/readfish#320] [General]
Hi, I would like to inquire about any suggestions you may have for what is the ideal desktop spec for optimal enrichment performance.  Currently we are considering: 
* Processor | Dual Intel® Xeon® Gold 5218R (27.5 MB cache, 20 cores, 40 threads, 2.10 GHz to 4.00 GHz Turbo, 125W)
* Graphics Card | Dual NVIDIA® RTX™ A4500, 20 GB GDDR6, NVLink SLI, 4 DP

Thanks for your help!

**Comment by mattloose** [comment]
This will be more than powerful enough.

Currently we are running on a single 48 core threadripper with two nvidia 4090 GPUs - we run these with one dedicated to MinKNOW for real time basecalling and the other one dedicated to running the adaptive sampling loop for readfish.
